img_proof/ipa_utils.py

```diff
@@ -195,30 +192,19 @@ def generate_instance_name(name):
 
 def generate_public_ssh_key(ssh_private_key_file):
```

Comment:
Should no longer be called "generate" as the code now reads a file and returns it's result, so it would be a "get" or "get_from_file"